src/projects/projects-page.html

```diff
@@ -94,23 +137,37 @@
       refurl="https://api.github.com/user/repos"
       last-response="{{projects}}"></github-authenticated-ajax>
 
-    <list-navigation
-        route="[[route]]" show-projects-list="{{showProjectsList}}"
-        show-pull-list="{{showPullList}}" selected-pull="[[selectedPull]]"
-        name-tail="{{nameTail}}" project="[[projectTitle]]"
-        projects="[[projects]]" pullrequests="[[pullrequests]]"
-        on-opened-project="_focusProject"></list-navigation>
-    <div class="filler"></div>
-    <iron-pages
-        class="project"
-        selected="[[_getCurrentView(projectSelected, pullRequestSelected)]]"
-        attr-for-selected="data-route"
-        on-tap="_tapHideProjectList">
-      <no-project-selected data-route="no-selected"></no-project-selected>
-      <project-overview-page data-route="overview" tabindex="0"
-          route="{{nameTail}}" pullrequests="[[pullrequests]]" github-user="[[githubUser]]"></project-overview-page>
-      <pull-request-page data-route="pr" selected-pull="[[selectedPull]]" route="{{prTail}}" github-user="[[githubUser]]"></pull-request-page>
-    </iron-pages>
+
+    <div class="paper-header">
+      <paper-tabs selected="{{page.page}}" scrollable attr-for-selected="data-route">
+        <paper-tab data-route="">
+          <a href="/">Home</a>
+        </paper-tab>
+        <paper-tab data-route="projects">
+          <a href$="[[route.prefix]]">My projects</a>
+        </paper-tab>
+      </paper-tabs>
+      <paper-button on-tap="_signout">Sign out</paper-button>
```

Comment:
We changed the class of the signout button with the on-tap, but the method itself has not changed it location.
Therefore now the signout button is broken.
We only cannot just change the class of the method, as this class does not has a fb instance.
Therefore we might need to add a listener, as is done with sign in.